src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/AbstractOperations.java

```diff
@@ -54,6 +55,29 @@ protected AbstractOperations(DSLContext create) {
     }
 
     /**
+     * executes the function if the experiment is not running.
+     * @param experimentID the id of the experiment
+     * @param function the function to execute
+     * @param <R> the return type
+     * @return the result of the function
+     */
+    protected <R> R doIfRunning(int experimentID, Function<Configuration, R> function) {
```

Comment:
thank you! the signature is right, doIfNotRunning is already existing and i copied the method and change the logic a bit